admin-manual/installation-setup/installation/_csv/install-landing.csv

```diff
@@ -0,0 +1,12 @@
+Intended Use,Environment,Related Documents,Notes
+Test-driving the latest release,Archivematica sandbox,:ref:`Using the sandbox <using-the-sandbox>`,"The sandbox will automatically reset daily. Any packages created will not be permanently saved. Additionally, there may be more than one demo user logged in at the same time, so you may see changes made by others while using the software."
+,Local virtual machine using Vagrant and VirtualBox,:ref:`Installing on a virtual machine using Vagrant <installing-on-vm>`,This virtual machine is not intended to be used in production.
+New installations – Production deployment,Ubuntu 14.04 64-bit Server Edition,:ref:`Automated install using Ansible \> Ubuntu 14.04 \(Trusty\)<ubuntu-14.04>`,"These instructions will not work if you are using a Windows machine as the host environment. For Windows installations, see the manual install instructions for production."
+,Ubuntu 16.04 64-bit Server Edition,:ref:`Automated install using Ansible \> Ubuntu 16.04 \(Xenial\)<ubuntu-16.04>`,"These instructions will not work if you are using a Windows machine as the host environment. For Windows installations, see the manual install instructions for production."
+,Ubuntu 18.04 64-bit Server Edition ( Experimental support ),:ref:`Automated install using Ansible \> Ubuntu 18.04 \(Bionic\)<ubuntu-18.04>`,Ansible playbooks for Ubuntu 18.04 are available but experimental.
+,CentOS 7 64-bit,:ref:`Manually Installing Archivematica on Ubuntu<install-pkg-centos>`,Archivematica versions 1.5.1 and higher support installation on CentOS/Red Hat.
```

Comment:
I think the title should be `Installing Archivematica on CentOS/Red Hat`.

Reply:
Updating per your suggestion.